common/artifacts/00_artifacts.txt

```diff
@@ -2298,8 +2284,8 @@ lance_tier_2_new = {
 	martial = 1
 	combat_rating = 7
 	command_modifier = {
-		cavalry = 0.05
-		pikemen = 0.10
+		light_cavalry = 0.05
+		knights = 0.10
```

Comment:
Then here must be not "knights = 0.10&light_cavalry = 0.05", but "light_cavalry = 0.10&knights = 0.05".
The same rule applies to lances below.